utils-core/main/default/lwc/datatable/datatable.js

```diff
@@ -768,7 +769,7 @@ export default class Datatable extends LightningElement {
     get containerClass() {
         let css = 'slds-border_top slds-border_bottom slds-border_left slds-border_right slds-is-relative ';
         if (this.useRelativeMaxHeight) {
-            css += 'table-vh ';
+            css += !!this.customRelativeMaxHeight ? '' : 'table-vh ';
```

Comment:
@jamessimone nice use of double bang here, it's not often I see others in the wild use it.

I also probably don't use it as much as I probably should.

Reply:
Thanks! It’s a good one. There was a joke at the React shop I was with for a while that the single bang wasn’t expressive enough - only double and triple would do it!